c7n/policy.py

```diff
@@ -826,6 +826,9 @@ def validate(self):
                 "Invalid Execution mode in policy %s" % (self.data,))
         m.validate()
         self.validate_policy_start_stop()
+        # support validation on k8s custom resource types
+        if hasattr(self.resource_manager, 'validate'):
```

Comment:
no hasattr please, extend the interface of resource manager with validate, and a doc string to clarify its what its not doing (mode, filters, actions)

Reply:
sounds good